crates/backend/src/local.rs

```diff
@@ -169,6 +203,11 @@ impl ReadBackend for LocalBackend {
 
         let walker = WalkDir::new(self.path.join(tpe.dirname()))
             .into_iter()
+            .inspect(|r| {
+                if let Err(err) = r {
+                    error!("Error while listing files: {err:?}");
+                }
+            })
```

Comment:
Yes, I think special error handling cases need to come a bit later, because otherwise this gets too big. I actually introduced this, because before we just threw away the error, doing nothing.